VideoWeb/VideoWeb/ClientApp/src/app/services/no-sleep-v2.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { Logger } from './logging/logger-base';
+
+import NoSleepApp from 'no-sleep-app';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class NoSleepServiceV2 {
+    noSleep = new NoSleepApp();
+
+    private loggerPrefix = '[NoSleepServiceV2] -';
+
+    constructor(private logger: Logger) {}
+
+    enable() {
+        console.log(`${this.loggerPrefix} no sleep status: ${this.noSleep.isEnabled}`);
```

Comment:
We're injecting the logger, but using console.log here

Reply:
ha needed to debug in the test, didn't think about that. I swear the lint should have picked it up